### ISSUE TITLE
BuildConfigurationTest uses deprecated ObjectMapper.configure method

### ISSUE THREAD

**Issue by manusa** [eclipse/jkube#2525]
### Description

The class [BuildConfigurationTest](https://github.com/eclipse/jkube/blob/275dc11e9bd0a268aac94f112ac2c443960d957b/jkube-kit/config/image/src/test/java/org/eclipse/jkube/kit/config/image/build/BuildConfigurationTest.java) uses a deprecated method.

The following lines should be changed to (updating imports if necessary):

https://github.com/eclipse/jkube/blob/275dc11e9bd0a268aac94f112ac2c443960d957b/jkube-kit/config/image/src/test/java/org/eclipse/jkube/kit/config/image/build/BuildConfigurationTest.java#L237-L238

- [ ] should be changed to:
```java
    final ObjectMapper mapper = JsonMapper.builder().configure(MapperFeature.USE_ANNOTATIONS, false).build();
```

### Before you start :red_circle:

:point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down: :point_down::point_down::point_down::point_down::point_down::point_down::point_down:
Make sure you read the [contributing guide](https://www.eclipse.org/jkube/contributing) first. Pay special attention to the ECA agreement section and the requirement to sign-off your commit.

#### Kubernetes

If you don't have a real Kubernetes cluster available (most probably), you can use Minikube or Kind to test with a local cluster.

- Minikube [get started guide](https://minikube.sigs.k8s.io/docs/start/)
- Kind (Requires Docker) [quick start](https://kind.sigs.k8s.io/docs/user/quick-start/)

#### OpenShift

If you don't have a real OpenShift cluster available (most probably), you can use [Red Hat's developer Sandbox for Red Hat OpenShift](https://developers.redhat.com/developer-sandbox). The only requirement is to have a Red Hat account.

Once you have your Sandbox environment, you'll need to download the oc tool from the cluster console.
(Press the `?` icon and from the context menu select `Command line tools`, you'll be redirected to https://$subdomain.openshiftapps.com/command-lines-tools where you'll be able to download the CLI for your platform)

**Comment by sankritimishra** [comment]
I'll be working on this

**Comment by anglerfishlyy** [comment]
can i work on this?

**Comment by rohanKanojia** [comment]
@anglerfishlyy : Are you still working on https://github.com/eclipse/jkube/pull/2879 ?

**Comment by anglerfishlyy** [comment]
No sir.
 I would like to go with first timers only issue.

**Comment by manusa** [comment]
Hi @anglerfishlyy, what's the status on this issue, have you progressed?

**Comment by anglerfishlyy** [comment]
actually , the lines are already changed to required one in the code. Is it still an issue that needs to be resolved?

**Comment by rohanKanojia** [comment]
@anglerfishlyy : Yes, you're right. Code still needs to be updated.

**Comment by manusa** [comment]
You're right, this was fixed in #2549

**Comment by manusa** [comment]
#2528 wasn't fixed, so maybe you can attempt to fix that one. @anglerfishlyy, if you want, please comment in #2528 so that we can assign it to you.
